src/components/Roadmap/Roadmap.jsx

```diff
@@ -24,11 +25,22 @@ const Roadmap = () => {
   const [user1Pic, setUser1Pic] = useState(user1Picture); //replace user1 with null instead of image placeholder
```

Comment:
What is this code? There's no setUser1Pic anywhere? Commenting because user1Pic and user2Pic is used later